server/utils/taskQueue.js

```diff
@@ -0,0 +1,38 @@
+const bull = require("bull");
+const { getAllCompanyName } = require("../models/user-model");
+const searchQueue = new bull("searchQueue", {
+  redis: { port: 6379, host: "127.0.0.1" },
+});
+const { redditSearch, getReddit } = require("../crawlers/reddit");
+const { getAndStoreTwitterData } = require("../crawlers/twitter");
+const User = require("../models/user-model");
+
+const companyQueue = new bull("companyQueue", {
```

Comment:
It is better to create a method that can create a bull queue and move new bull line into that function.